frameworks/symfony

```diff
@@ -49,19 +59,29 @@ case "$1" in
         ;;
     compile)
         echo "-----> Setting up symfony app"
-        cp "$basedir/../conf/nginx/symfony.conf.erb" "$BUILD_DIR/conf/site.conf.erb"
+        if requires_symfony_flex; then
+          cp "$basedir/../conf/nginx/symfony4.conf.erb" "$BUILD_DIR/conf/site.conf.erb"
+        else
+          cp "$basedir/../conf/nginx/symfony.conf.erb" "$BUILD_DIR/conf/site.conf.erb"
+        fi
         ;;
     end)
         BUILD_DIR="$1"
         CACHE_DIR="$2"
         APP_DIR="/app"
 
-        status "Warmuping cache"
-        rm -rf app/cache/*
+        status "Warmping cache"
 
         workdir=$(get_workdir)
         console=$(get_console)
-        php $console cache:warmup --env=prod --no-debug | indent
+        rm -rf $workdir/cache/*
+        if is_symfony4 ; then
```

Comment:
Why this? the `$console` variable should contain `bin/console` or `app/console` according to the version of symfony

Reply:
Well yep but with `php $console [...]` the script return:

```
2018-01-08T15:59:36+00:00 [error] Error thrown while running command "'bin/console' 'cache:warmup' --env=prod --no-debug". Message: "Command "bin/console" is not defined."                                                                                                                                                                                                                   
                                                                                                                                                                                                                                                                                                                                                                                              
                                                                                                                                                                                                                                                                                                                                                                                              
  Command "bin/console" is not defined.                                                                                                                                                                                                                                                                                                                                                       
```

They must use $0 wrongly

Reply:
Please let a comment then

Reply:
Already done